deployment/helmfile.yaml

```diff
@@ -21,5 +22,5 @@ releases:
           tag: {{ trimPrefix "refs/tags/" (requiredEnv "IMG_TAG") }}
           repository: {{ .Namespace }}/cloud-portal
       - fullnameOverride: {{ env "HELM_RELEASE_NAME" | default "portal" }}
-      - podAnnotations:
+        podAnnotations:
           app.kubernetes.io/git-shasum: {{ env "GIT_SHA" | default "GIT_SHA" }}
```

Comment:
This seems a little inconsistent. If I understand the `helmfile` correctly it will merge these inline value list items? I would propose  one list item per key or one list item for all inline values.

Reply:
Fine by me, but It doesn't actually matter